packages/preferences/src/browser/views/preference-widget.tsx

```diff
@@ -50,6 +50,18 @@ export class PreferencesWidget extends Panel implements StatefulWidget {
         return this.tabBarWidget.currentScope;
     }
 
+    get availableTitles(): Title.Dataset[] {
+        return this.tabBarWidget.availableTitles;
+    }
+
+    openWithScope(selection: Preference.SelectedScopeDetails): void {
```

Comment:
minor: possibly rename the method?

```suggestion
    setScopeSelection(selection: Preference.SelectedScopeDetails): void {
```